test/spec/functional/listView.js

```diff
@@ -237,28 +241,28 @@ describe('A List View', function() {
     var model3 = new Model({order: 1});
     myCollection.add(model3);
     expect(myListView.$el.find('div.item').length).toBe(3);
-    expect(_.first(myListView.$el.children())).toBe(myListView.getChildView(model).el);
-    expect(_.last(myListView.$el.children())).toBe(myListView.getChildView(model3).el);
+    expect(_.first(myListView.$el.children())).toBe(myListView.getChildViewFromModel(model).el);
+    expect(_.last(myListView.$el.children())).toBe(myListView.getChildViewFromModel(model3).el);
     myCollection.comparator = 'order';
     myCollection.sort();
-    expect(_.first(myListView.$el.children())).toBe(myListView.getChildView(model3).el);
-    expect(_.last(myListView.$el.children())).toBe(myListView.getChildView(model).el);
+    expect(_.first(myListView.$el.children())).toBe(myListView.getChildViewFromModel(model3).el);
+    expect(_.last(myListView.$el.children())).toBe(myListView.getChildViewFromModel(model).el);
   });
 
   it('can sort a large list in a reasonable time', function() {
     var startTime, endTime, i,
         numberOfViews = 1000,
-        threshold = 300;
+        threshold = 500;
```

Comment:
This test is always awkward...  Ideally it should be machine agnostic...  We might need to think of a more cross-cpu-friendly way of conducting this test.